lib/chitragupta/cls.js

```diff
@@ -0,0 +1,26 @@
+const CLS = require('cls-hooked');
+const CONSTANTS = require('./constants');
+
+const NS = CLS.createNamespace(CONSTANTS.CHITRAGUPTA_CLS_ID);
+
+function get(key) {
+  let returnVal = null;
+  if (NS && NS.active) {
+    returnVal = NS.get(key);
+  }
+  return returnVal;
+}
+
+function set(key, value) {
+  let returnVal = null;
+  if (NS && NS.active) {
+    returnVal = NS.set(key, value);
+  }
+  return returnVal;
```

Comment:
Why is `set` returning a value?

Reply:
By default `set` function of the cls namespace returns true false. Just wanted to ensure to return it back